cmd/argocd/commands/app.go

```diff
@@ -316,6 +316,17 @@ func printHeader(acdClient argocdclient.Client, app *argoappv1.Application, ctx
 	}
 }
 
+// getSourceNameToPositionMap returns a map of source name to position
+func getSourceNameToPositionMap(app *argoappv1.Application) map[string]int64 {
+	sourceNameToPosition := make(map[string]int64)
+	for i, s := range app.Spec.Sources {
+		if s.Name != "" {
+			sourceNameToPosition[s.Name] = int64(i + 1)
```

Comment:
It's a bit confusing that we use indexes starting from 1, but this is out of scope of this PR.

Reply:
@andrii-korotkov-verkada , We do not consider this to be an index, but the positions. There was a discussion in the community around this and we agreed to start with 1 on CLI and 0 on apis.

Postion = 1 --> Index = 0.